rbac/rbac/urls.py

```diff
@@ -37,6 +37,7 @@
 urlpatterns = [
     re_path(r"^{}v1/".format(API_PATH_PREFIX), include("api.urls")),
     re_path(r"^{}v1/".format(API_PATH_PREFIX), include("management.urls")),
+    re_path(r"^{}v2/".format(API_PATH_PREFIX), include("management.v2_urls")),
```

Comment:
Out of curiosity, were you involved in some of the initial discussion around standards for versioning APIs on the platform? I thought so, but couldn't remember where that all landed.

Reply:
I couldn't remember as well and I also did not find anything regard this. Let me know if you will find something!